packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -275,10 +297,10 @@ const DropdownBase = kind({
 	},
 
 	render: ({children, disabled, onOpen, onSelect, open, selected, width, title, ...rest}) => {
-		const popupProps = {children, onSelect, selected, width};
+		const popupProps = {children, onSelect, selected, width, role: ''};
 
 		// `ui/Group`/`ui/Repeater` will throw an error if empty so we disable the Dropdown and prevent Dropdown to open if there are no children.
-		const hasChildren = children && children.length;
+		const hasChildren = filterChildren(children).length;
```

Comment:
Why don't we use computed here?  No real reason to change it, I guess.  Just curious.

Reply:
Actually don't need to do it here. It can't be computed because `children` is computed and the computations are independent. However, _because_ `children` is computed and filters the incoming value, we need not do so here.

Reply:
Both good points.